hax-lib/macros/src/dummy.rs

```diff
@@ -76,6 +76,11 @@ pub fn lemma(_attr: TokenStream, _item: TokenStream) -> TokenStream {
     quote! {}.into()
 }
 
+#[proc_macro_attribute]
+pub fn proof(_attr: TokenStream, _item: TokenStream) -> TokenStream {
+    quote! {}.into()
+}
```

Comment:
This will drop the item, I don't think that's what you want.
You should just add `proof` in `identity_proc_macro_attribute!{...}`.

Also, please do not name it `proof`, but `lean_proof`, and then expose it under `hax_lib::lean::proof`